synapse/events/__init__.py

```diff
@@ -475,4 +500,4 @@ def make_event_from_dict(
 ) -> EventBase:
     """Construct an EventBase from the given event dict"""
     event_type = event_type_from_format_version(room_version.event_format)
```

Comment:
This is now the only use of `event_type_from_format_version`, should it be considered "private" (prefixed with a `_`)?

Reply:
probably. done.